### PR TITLE
Swagger specs for all existing versions of Microsoft.PolicyInsights

### PR DESCRIPTION
**Pull request by bulentelmaci**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by salameer** [comment]
Re-assigning to balance efforts

**Comment by bulentelmaci** [comment]
@sarangan12 How can I suppress EQUIVALENT_PATH errors I am getting? We need those paths separated because semantically they are different for us. If we merge them, APIs won't be useful at all.
I see that there are other RPs using the same pattern in some paths - not sure how they suppressed the errors. Any guidance you can provide?

**Comment by sarangan12** [comment]
Please fix this error: https://travis-ci.org/Azure/azure-rest-api-specs/jobs/344500200#L2257

**Comment by bulentelmaci** [comment]
Sent an e-mail to spec violations review committee about the suppressions in readme.md asking for approval:

•	EnumInsteadOfBoolean in the definitions
We have a property defined in two different definitions as Boolean. Source of these properties is our Kusto data store, and the data type in that store is boolean. If we change the type in RP, it will break existing clients, and we will incur runtime conversion cost.
•	NonApplicationJsonType in two specific paths
These are $metadata endpoints. They return ODATA EDM metadata document in XML format which ODATA client generators consume. Only those paths are marked to produce application/xml

**Comment by sarangan12** [comment]
Please take a look at the errors reported here: https://travis-ci.org/Azure/azure-rest-api-specs/jobs/345496880

**Comment by bulentelmaci** [comment]
All errors in that job belong to other RPs. The ones I am checking in are clean:


    ✓ /home/travis/build/Azure/azure-rest-api-specs/specification/policyinsights/resource-manager/Microsoft.PolicyInsights/PolicyEvents/preview/2017-08-09-preview/policyEvents.json should be semantically valid. (150ms)

info: No Errors were found.

    ✓ /home/travis/build/Azure/azure-rest-api-specs/specification/policyinsights/resource-manager/Microsoft.PolicyInsights/PolicyEvents/preview/2017-10-17-preview/policyEvents.json should be semantically valid. (74ms)

info: No Errors were found.

    ✓ /home/travis/build/Azure/azure-rest-api-specs/specification/policyinsights/resource-manager/Microsoft.PolicyInsights/PolicyEvents/preview/2017-12-12-preview/policyEvents.json should be semantically valid. (114ms)

info: No Errors were found.

    ✓ /home/travis/build/Azure/azure-rest-api-specs/specification/policyinsights/resource-manager/Microsoft.PolicyInsights/PolicyStates/preview/2017-08-09-preview/policyStates.json should be semantically valid. (133ms)

info: No Errors were found.

    ✓ /home/travis/build/Azure/azure-rest-api-specs/specification/policyinsights/resource-manager/Microsoft.PolicyInsights/PolicyStates/preview/2017-10-17-preview/policyStates.json should be semantically valid. (73ms)

info: No Errors were found.

    ✓ /home/travis/build/Azure/azure-rest-api-specs/specification/policyinsights/resource-manager/Microsoft.PolicyInsights/PolicyStates/preview/2017-12-12-preview/policyStates.json should be semantically valid. (189ms)

info: No Errors were found.

**Review by sarangan12 — APPROVED** [review]

**Comment by sarangan12** [comment]
Once I get the suppression approval, I will merge the PR. @bulentelmaci Do you have any concerns with me merging the PR?

**Comment by bulentelmaci** [comment]
@sarangan12 No concerns.

**Comment by sarangan12** [comment]
Update: Samer has provided the suppression approval.

File: specification/policyinsights/resource-manager/Microsoft.PolicyInsights/PolicyStates/preview/2017-08-09-preview/policyStates.json
Comment (ravbhatnagar) [inline specification/policyinsights/resource-manager/Microsoft.PolicyInsights/PolicyStates/preview/2017-08-09-preview/policyStates.json]: You dont need this in both the files. This operation is needed per RP namespace.

**Comment by ravbhatnagar** [comment]
Looks good. Just one minor note added. Also asked Bulent to schedule something with Azure API review board. But this is good to merge to unblock public preview.

File: specification/policyinsights/resource-manager/Microsoft.PolicyInsights/PolicyStates/preview/2017-08-09-preview/policyStates.json
Comment (sarangan12) [reply specification/policyinsights/resource-manager/Microsoft.PolicyInsights/PolicyStates/preview/2017-08-09-preview/policyStates.json]: @bulentelmaci Can you fix this? Seems to be a quick fix

Comment (bulentelmaci) [reply specification/policyinsights/resource-manager/Microsoft.PolicyInsights/PolicyStates/preview/2017-08-09-preview/policyStates.json]: @sarangan12 Working on it

Comment (bulentelmaci) [reply specification/policyinsights/resource-manager/Microsoft.PolicyInsights/PolicyStates/preview/2017-08-09-preview/policyStates.json]: @sarangan12 Pushed changes.

**Pull request merged**
